extmod/modussl_axtls.c

```diff
@@ -167,10 +167,16 @@ STATIC mp_obj_ssl_socket_t *ussl_socket_new(mp_obj_t sock, struct ssl_args *args
         o->ssl_sock = ssl_client_new(o->ssl_ctx, (long)sock, NULL, 0, ext);
 
         if (args->do_handshake.u_bool) {
-            int res = ssl_handshake_status(o->ssl_sock);
-
-            if (res != SSL_OK) {
-                ussl_raise_error(res);
+            int r = ssl_handshake_status(o->ssl_sock);
+
+            if (r != SSL_OK) {
+                ssl_display_error(r);
+                if (r == SSL_CLOSE_NOTIFY || r == SSL_ERROR_CONN_LOST) { // EOF
```

Comment:
@tve to get the `tests/extmod/ussl_basic.py` test to pass on unix, this line should be:
```
if (r == SSL_CLOSE_NOTIFY) {
```
But I'm not sure if that's exactly correct.

Reply:
I'll have to look...

Reply:
I removed the check for `SSL_ERROR_CONN_LOST`.  Doing this, as far as I can tell it doesn't make axtls any better or worse at the net_inet/net_hosted tests, and means it still passes the main test suite.  We can fix axtls further in subsequent PRs.